community/io/src/main/java/org/neo4j/io/pagecache/PageCache.java

```diff
@@ -61,6 +59,24 @@
      */
```

Comment:
Previous line in java doc mention exclusive mapping conflicts - those are gone with this PR, so the reason for IOException should be updated as well